src/openms/source/CHEMISTRY/ISOTOPEDISTRIBUTION/IsotopeDistribution.cpp

```diff
@@ -108,6 +108,10 @@ namespace OpenMS
     return distribution_[0].getMZ();
   }
 
+  Peak1D IsotopeDistribution::getMostAbundant() const
+  {
+      return *std::max_element(begin(), end(), MassAbundance::IntensityLess());
```

Comment:
you may have to check here that its not empty, otherwise you return `last()` which will lead to segfaults and issues in the Python bindings, see https://en.cppreference.com/w/cpp/algorithm/max_element

> Iterator to the greatest element in the range [first, last). If several elements in the range are equivalent to the greatest element, returns the iterator to the first such element. Returns last if the range is empty.